packaging/opensuse/snapd.spec

```diff
@@ -272,11 +272,12 @@ done
 %install
 # Install all systemd and dbus units, and env files.
 %make_install -C %{indigo_srcdir}/data \
-		BINDIR=%{_bindir} \
-		LIBEXECDIR=%{_libexecdir} \
-		DATADIR=%{_datadir} \
-		SYSTEMDSYSTEMUNITDIR=%{_unitdir} \
-		SNAP_MOUNT_DIR=%{snap_mount_dir}
+        BINDIR=%{_bindir} \
```

Comment:
@sergiocazzolato did tabs changed to spaces or back in this edit? It looks suspicious.

Reply:
@zyga I just moved tabs to spaces because the file contains spaces but in that line there were tabs.

Reply:
It was to keep consistency in the file